document-service/doc-api/src/doc_api/utils/base.py

```diff
@@ -0,0 +1,51 @@
+# Copyright © 2021 Province of British Columbia
```

Comment:
i'm probably going to pull this into an installable, will poke you about it.

Reply:
Copy that.